doc: Overview diagram, and some doc additions (#976)
akka/akka-projection: akka-edge-docs/src/main/paradox/feature-summary.md

### Edge is producer

> This cases describe when the edge service is the producer of events.

Reviewer: Same fix as above?

```suggestion
This case describes when the edge service is the producer of events.
```

Or `These cases describe` for plural.

Author: thanks, s at wrong word

H2 database should not be used when the service is an Akka Cluster with more than 1 node.
@@@

> ## Replicated Event Sourcing is not for Edge

Author: @johanandren what do you think about this?

@extref[Replicated Event Sourcing over gRPC](akka-distributed-cluster:feature-summary.html#replicated-event-sSourcing-over-grpc)
is a useful feature in Akka Distributed Cluster, but it is not recommended for edge use cases. The reasons why it is currently
not supported for Akka Edge are:

* It requires gRPC connectivity in both directions between the replicas.

> * The overhead of CRDT metadata may become to large when there are many 100s of replicas, or if the replicas dynamically change over time.

Reviewer: ```suggestion
* The overhead of CRDT metadata may become too large when there are many 100s of replicas, or if the replicas dynamically change over time.
```

That said, if you can overcome these restrictions it can be a good fit also for edge use cases. You might have 

> a network topology that allow establishing connections in both directions (e.g. VPN solution) and you might not have

Reviewer: ```suggestion
a network topology that allows establishing connections in both directions (e.g. VPN solution) and you might not have
```
